src/app/views/initial-setup/initial-setup-name/initial-setup-name.component.ts

```diff
@@ -14,6 +14,7 @@ export class InitialSetupNameComponent implements OnInit {
 
```

Comment:
how is the username set into the dataservice?

Reply:
handled by the setup nav component

Reply:
could this be moved to the initial-setup-name.component.ts? it should work especially if you use a normal input. (add a component variable 'username', bind with [(ngModel)] to input field and add a (change) event listener)

Reply:
i'll try